src/Domain/CoordinateMaps/TimeDependent/ShapeMapTransitionFunctions/Wedge.cpp

```diff
@@ -5,139 +5,416 @@
 
 #include <array>
 #include <cmath>
+#include <limits>
 #include <optional>
 #include <pup.h>
+#include <pup_stl.h>
+#include <type_traits>
 
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
 #include "Domain/CoordinateMaps/TimeDependent/ShapeMapTransitionFunctions/ShapeMapTransitionFunction.hpp"
 #include "Domain/Structure/Direction.hpp"
 #include "NumericalAlgorithms/RootFinding/QuadraticEquation.hpp"
+#include "Utilities/Algorithm.hpp"
 #include "Utilities/ConstantExpressions.hpp"
 #include "Utilities/ContainerHelpers.hpp"
 #include "Utilities/EqualWithinRoundoff.hpp"
 #include "Utilities/ErrorHandling/Assert.hpp"
 #include "Utilities/ErrorHandling/CaptureForError.hpp"
 #include "Utilities/ErrorHandling/Error.hpp"
 #include "Utilities/MakeArray.hpp"
+#include "Utilities/MakeString.hpp"
 #include "Utilities/MakeWithValue.hpp"
 #include "Utilities/StdArrayHelpers.hpp"
 #include "Utilities/StdHelpers.hpp"
 
+namespace {
+template <typename T>
+int sgn(T val) {
+  return (T(0) < val) - (val < T(0));  // NOLINT
```

Comment:
What's wrong with the implementation in `src/Utilities/Math.hpp`?

Reply:
Just that I didn't know it existed. I'll use that instead